.gitignore

```diff
@@ -1,2 +1,2 @@
-stmp.toml
```

Comment:
prefer to keep this as it prevents accidental upload of the real config with credentials